### DISCUSSION TITLE
clipboard-manager

### DISCUSSION THREAD

**Discussion by Stradios** [cosmic-utils/clipboard-manager#92] [Ideas]
Works good, i  like it.
I don't know this if I can do this, but a nice feather would be to have a command to open the interface of the app to add in the keybinding for fast switching

**Comment by wiiznokes** [comment]
Yeah, i plan to implement that soon (tm) (see https://github.com/cosmic-utils/clipboard-manager/issues/20), but there are still some unresolved questions, and i need to test it with multiple monitor so i can't rn

**Comment by guicosena** [comment]
May I suggest 2 features?
1 - A Clean All button
2 - A button to copy the text on the clipboard directly to the text field instead of selecting the clip and using ctrl+v.

**Reply by wiiznokes** [reply]
There is already a clean all button when you right click the icon of the applet. 
For the second feature, i don't think there is a way to do this only using the standard wayland protocol. Do you know a clipboard manager that does it in wayland ?

**Reply by guicosena** [reply]
1 - Oh! I missed that. Sorry
2 - I don't know if the clipboard indicator use a wayland protocol (https://github.com/Tudmotu/gnome-shell-extension-clipboard-indicator), but it has the feature that I talked about and it only works on the most recent version of gnome.
<img width="637" height="472" alt="Captura de tela de 2025-09-09 09-50-01" src="https://github.com/user-attachments/assets/ca115a6b-d178-4211-92e8-4dc05838cd62" />

I'm not a developer, I just wanted to suggest the features that I missed the most.
